### ISSUE TITLE
Is this project still active?

### ISSUE THREAD

**Issue by rbunn80110** [ai-fast-track/timeseries#8]
Just curious if this project will continue going forward?

**Comment by vrodriguezf** [comment]
I don't think so. I forked it just to update it for the new fastai release, in case you want to use it
